src/app/core/components/tab/tab.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-tab',
+  templateUrl: './tab.component.html',
+  styleUrls: ['./tab.component.scss'],
+})
+export class TabComponent {
+  @Input() types: string[] = [];
```

Comment:
i don't think types is meaningful enough. Maybe something like `items` or `tabs`

Reply:
tabs and currentTab